contracts/solidity/Markets/DefaultMarket.sol

```diff
@@ -110,7 +112,13 @@ contract DefaultMarket is Market {
         // Buy all outcomes
         eventContract.buyAllOutcomes(outcomeTokenCosts);
         // Transfer outcome tokens to buyer
-        eventContract.outcomeTokens(outcomeTokenIndex).transfer(msg.sender, outcomeTokenCount);
+        require(eventContract.outcomeTokens(outcomeTokenIndex).transfer(msg.sender, outcomeTokenCount));
```

Comment:
@collinc97 So sorry, just got the message from Stefan: `transfer` should return a success code instead of revert. I think that is the way to go, but that also means that this change is necessary. I think he has already changed the success code return, but yeah, it should be changed back 🤓